packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -289,17 +300,48 @@ const styles: ComponentStyles<ISiteManagedClasses, IDevSiteDesignSystem> = {
     }
 };
 
+export interface ILocale {
+    displayName: string;
+    direction: Direction;
+}
+
+export interface ILocaleMapping {
+    en: ILocale;
+    [locale: string]: ILocale;
+}
+
 class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClasses>, ISiteState> {
 
     public static defaultProps: Partial<ISiteProps> = {
-        locales: ["en", "en-rtl"]
+        locales: {
+            en: {
+                displayName: "English",
+                direction: Direction.ltr,
+            },
+            "en-rtl": {
+                displayName: "English-RTL",
+                direction: Direction.rtl
+            }
+        }
```

Comment:
I believe that the locale changes here will be a breaking change for any users who currently use the dev site.

Is there a reason we are including these changes in this PR instead of in a different one?

Reply:
@chrisdholt The issues Jason put in the One Note was the styling of the select to match the design and 'en' should be 'English'

Reply:
Sure - this is a breaking change though. Have you booted this up in the MSFT package to ensure that it works there without changing that package configuration? I think it's going to break.

Reply:
I see, it's not breaking currently in MSFT because we are relying on the default props. However, we've changed from a "string" to an interface with an additional key for the props. It isn't breaking our MSFT scenario but I believe this is a breaking change. We can still move forward with it, but I'm more curious why we made these additional changes. It seems to me that we could technically just change "en" to "English" in the MSFT implementation to solve for this.

Reply:
Primarily, just offering some insight to consider. I think in this specific case though we ended up changing the entire fundamentals of how the thing works and the stated purpose of that change doesn't map to the issue or PR title. So, looking at this, it seems like the larger refactor which would've been better suited to it's own PR. The breaking change itself doesn't make sense in this context to me, and while I think it's the correct code solution, this will get lost in the changelog as part of this PR/title.